engine/src/test/scala/cromwell/engine/backend/local/LocalBackendSpec.scala

```diff
@@ -33,45 +23,43 @@ class LocalBackendSpec extends CromwellTestkitSpec with Mockito with WorkflowDes
-
-  val prefix = "this_is_a_prefix"
-  val cwd = Paths.get(".")
-
-  def testFailOnStderr(descriptor: OldStyleWorkflowDescriptor, expectSuccess: Boolean): Unit = {
-    val call = descriptor.namespace.workflow.calls.head
-    val jobDescriptor = OldStyleBackendCallJobDescriptor(descriptor, BackendCallKey(call, None, 1), Map.empty[String, WdlValue])
-    jobDescriptor.execute map { _.result } map {
-      case OldStyleNonRetryableFailedExecution(e, _, _) => if (expectSuccess) fail("A call in a failOnStderr test which should have succeeded has failed ", e)
-      case OldStyleRetryableFailedExecution(e, _, _) => if (expectSuccess) fail("A call in a failOnStderr test which should have succeeded has failed ", e)
-      case OldStyleSuccessfulBackendCallExecution(_, _, _, _, _) => if (!expectSuccess) fail("A call in a failOnStderr test which should have failed has succeeded")
-      case OldStyleSuccessfulFinalCallExecution => fail("A FinalCall shouldn't have run in this failOnStderr")
-      case OldStyleAbortedExecution => fail("Not expecting this at all")
-    }
-  }
+  val testWorkflowManagerSystem = new TestWorkflowManagerSystem
 
   "LocalBackend" should {
-    "allow stdout if failOnStderr is set" ignore {
-      testFailOnStderr(stdoutDescriptor("runtime {failOnStderr: true}"), expectSuccess = true)
+    "allow stdout if failOnStderr is set" in {
+      runWdl(
+        sampleWdl = StdoutWdl,
+        eventFilter = EventFilter.info(pattern = "Workflow complete", occurrences = 1),
+        runtime = "runtime {failOnStderr: true}",
+        terminalState = WorkflowSucceeded )
     }
 
-    "not allow stderr if failOnStderr is set" ignore {
-      testFailOnStderr(stderrDescriptor("runtime {failOnStderr: true}"), expectSuccess = false)
+    "not allow stderr if failOnStderr is set" in {
+      runWdl(
+        sampleWdl = StderrWdl,
+        eventFilter = EventFilter.info(pattern = "transitioning from ExecutingWorkflowState to WorkflowFailedState", occurrences = 1),
+        runtime = "runtime {failOnStderr: true}",
+        terminalState = WorkflowFailed)
     }
 
-    "allow stdout if failOnStderr is not set" ignore {
-      testFailOnStderr(stdoutDescriptor("runtime {failOnStderr: false}"), expectSuccess = true)
+    "allow stdout if failOnStderr is not set" in {
+      runWdl(
+        sampleWdl = StdoutWdl,
+        eventFilter = EventFilter.info(pattern = "Workflow complete", occurrences = 1),
+        runtime = "runtime {failOnStderr: false}",
+        terminalState = WorkflowSucceeded )
     }
 
-    "allow stderr if failOnStderr is not set" ignore {
-      testFailOnStderr(stderrDescriptor("runtime {failOnStderr: false}"), expectSuccess = true)
+    "allow stderr if failOnStderr is not set" in {
+      runWdl(
+        sampleWdl = StderrWdl,
+        eventFilter = EventFilter.info(pattern = "Workflow complete", occurrences = 1),
+        runtime = "runtime {failOnStderr: false}",
+        terminalState = WorkflowSucceeded )
```

Comment:
There is an extra space before the closing `)` in the multiple `WorkflowSucceeded )` only. The above `WorkflowFailed)` is fine.